postgres/datadog_checks/postgres/query_count_cache.py

```diff
@@ -0,0 +1,36 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from cachetools import TTLCache
+
+# 8 hours
+CACHE_TTL = 60 * 60 * 8
```

Comment:
what's the reason behind 8 hours TTL?

Reply:
I was thinking "once per work day" since this data shouldn't be changing all that much. But I don't have a strong opinion on what it should be.